kanalony-storage-client/src/main/scala/kanalony/storage/generator/TablesMetadata.scala

```diff
@@ -0,0 +1,19 @@
+package kanalony.storage.generator
+
+/**
+ * Created by elad.benedict on 2/8/2016.
+ */
+
+object TablesMetadata {
+  val metadata = List(
+                    TableMetadata("hourly_user_activity_prtn_entry_clst_country",
+                        PrimaryKey(PartitionKey(List(ColumnDefinition("partner_id", ColumnType.Int),
+                                                     ColumnDefinition("entry_id", ColumnType.String),
+                                                     ColumnDefinition("event_type", ColumnType.Int),
```

Comment:
@olampert will change the "event_type" column name to **metric** and the "count" column name to **value**, unless you have a better suggestion...